src/pdf.scripting.js

```diff
@@ -22,4 +22,4 @@ const pdfjsVersion =
 const pdfjsBuild =
   typeof PDFJSDev !== "undefined" ? PDFJSDev.eval("BUNDLE_BUILD") : void 0;
 
-export { initSandbox };
+globalThis.pdfjsScripting = { initSandbox };
```

Comment:
Should we perhaps add a small comment here why we can't use `export` here? Looking at the code it would seem expected to do so here, so it might save the next person some time figuring out why `export` doesn't work here.